### PR TITLE
added quote

### PR DESCRIPTION
**Pull request by srosebattles**
Added a quote

### CODE REVIEW

**Comment by Demo318** [comment]
Hey, @srosebattles, can you change the 'URL' in your quote to point to Lewis' book on amazon?

File: js/index.js [hunk 1]
```diff
@@ -59,7 +59,19 @@ var quotes = [{
     "author": "Edwin H. Friedman",
     "source": "A Failure of Nerve: Leadership in the Age of the Quick Fix",
     "URL": "http://amzn.to/29ouuHs",
-              }
+  },
+  // {
+  //   "quote": "Friendship is born at that moment when one person says to another: What, you too? I thought I was the only one.",
+  //   "author": "C.S.Lewis",
+  //   "source": "The Four Loves",
+  //   "URL": "http://amzn.to/29ouuHs",
```
Comment (Demo318) [inline js/index.js]: Needs to point to CS Lewis' book.

**Comment by srosebattles** [comment]
kdoke

File: js/index.js [hunk 2]
```diff
@@ -59,7 +59,19 @@ var quotes = [{
...
+  // },
+  {
+    "quote": "Management is doing things right; Leadership is doing the right things",
+    "author": "Peter Drucker",
+    "source": "",
```
Comment (Demo318) [inline js/index.js]: Requires source & URL

**Pull request closed without merging**